src/components/App.js

```diff
@@ -40,6 +41,12 @@ class App extends Component<Props,State> {
     );
   }
 
+  componentDidUpdate(prevProps: Props, prevState: State) {
+    if (prevProps.isFullscreen && !this.props.isFullscreen) {
+      this.store.dispatch(start());
+    }
+  }
+
```

Comment:
This detects the props change on exiting fullscreen mode, converting it into a START action.